notebooks/community/model_garden/model_garden_pipeline_templates_bert.ipynb

```diff
@@ -0,0 +1,852 @@
+{
```

Comment:
<p>Do we also want to clean up the repo in artifact registry?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1815/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>